src/main/java/Duke.java

```diff
@@ -1,10 +1,233 @@
+        taskList[count] = task;
+        ++count;
+        getMessageForTodo(task, count);
+    }
+
+    private static void getMessageForTodo(Task task, int count) {
+        printHorizontalLine();
+        showAddTask();
+        System.out.println("  " + task.toString());
+        System.out.println("Now you have " + count + " tasks in the list.");
+        printHorizontalLine();
+    }
+
+    public static void showAddTask() {
+        System.out.println("Got it. I've added this task:");
+    }
+
+    /**
+     * Add task under Deadline class and feedback display message when Deadline task added
+     * @param commandArgs description and deadline of the task
+     */
+    private static void executeDeadline(String commandArgs) {
+        final String[] descriptionAndDeadline = splitDescriptionAndDeadline(commandArgs);
+        Task task = new Deadline(descriptionAndDeadline[0], descriptionAndDeadline[1]);
+        taskList[count] = task;
+        ++count;
+        getMessageForDeadline(task, count);
+    }
+
+
+    private static String[] splitDescriptionAndDeadline(String commandArgs) {
+        final String[] split = commandArgs.split(" /by ");
+        return split;
+    }
+
+    private static void getMessageForDeadline(Task task, int count) {
+        printHorizontalLine();
+        showAddTask();
+        System.out.println("  " + task.toString());
+        System.out.println("Now you have " + count + " tasks in the list.");
+        printHorizontalLine();
+    }
+
+    /**
+     * Add task under Event class and feedback display message when Event task added
+     * @param commandArgs
+     */
+    private static void executeEvent(String commandArgs) {
+        final String[] descriptionAndTime = splitDescriptionAndTime(commandArgs);
+        Task task = new Event(descriptionAndTime[0], descriptionAndTime[1]);
+        taskList[count] = task;
+        ++count;
+        getMessageForEvent(task, count);
+    }
+
+    private static String[] splitDescriptionAndTime(String commandArgs) {
+        final String[] split = commandArgs.split(" /at ");
+        return split;
+    }
+
+    private static void getMessageForEvent(Task task, int count) {
+        printHorizontalLine();
+        showAddTask();
+        System.out.println("  " + task.toString());
+        System.out.println("Now you have " + count + " tasks in the list.");
+        printHorizontalLine();
+    }
+
+    /**
+     * Generate feedback for list of task and task status
+     */
+    private static void executeList() {
+        printHorizontalLine();
+        showTasksList(taskList);
+        printHorizontalLine();
+    }
+
+    private static void showTasksList(Task[] taskList) {
+        System.out.println("Here are the tasks in your list:");
+        int descriptionCount = 1;
+        for (Task task : taskList) {
+            if (task != null) {
+                System.out.println(descriptionCount + "." + task.toString());
+                ++descriptionCount;
```

Comment:
As this will show the Task status and Task type as well, should descriptionCount be named more intuitively?